tools/mothballer.py

```diff
@@ -101,14 +102,14 @@ def build_category(categorydir, outdir):
         # open and gather some state
         existing = zipfile.ZipFile(zipfilename, 'r')
         try:
-            category_seed = existing.open(seedfn).read().strip()
-        except:
+            category_seed = existing.open(SEEDFN).read().strip()
```

Comment:
How could this have worked before? Is there a `seedfn` anywhere else in this file?

Reply:
Nope. Nowhere.